dipy/core/sphere.py

```diff
@@ -540,9 +540,8 @@ def interp_rbf(data, sphere_origin, sphere_target,
     from scipy.interpolate import Rbf
 
     def angle(x1, x2):
-        xx = np.arccos((x1 * x2).sum(axis=0))
-        xx[np.isnan(xx)] = 0
-        return xx
+        xx = np.arccos(np.clip((x1 * x2).sum(axis=0), 0, 1))
```

Comment:
`np.clip` should be between `-1,1` instead of `0,1`. It should work correctly after this change. But there is still something which disturbs me from my previous comment.